client/src/components/categorical/value.js

```diff
@@ -105,6 +106,22 @@ class CategoryValue extends React.Component {
       categories = schema.annotations.obsByName[colorAccessor]?.categories;
     }
 
+    const longLength = 35;
```

Comment:
These should be more descriptively named and moved to globals: 

`globals.categoryLabelDisplayStringMaxLength` or something similar